Add new test scenario for installation from marketplace

apodhrad opened the pull request.

mickaelistria commented:
Why did you have to add an additional click on "Next >" for all the scenarios? I don't have the impression that a ne wpage was introduced in all install wizards.

apodhrad commented:
Because I removed it from InstallTest.continueInstall [1]. Not all installation scenarios (such as marketplace installation) allows to click 'Next >' so I moved this step to each test where it is needed.

[1] https://github.com/jbosstools/jbosstools-install-grinder/pull/28/files#diff-4350d9c09d44b1603940007792fbb014L152

mickaelistria commented:
Ok. thanks. Merging.
